From 6673b9b1407ab211c1682f4fe386c5efed4a896b Mon Sep 17 00:00:00 2001
From: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
Date: Wed, 11 Sep 2019 11:10:41 +0530
Subject: [PATCH] Fix for VTS test SupplicantStaIfaceHidlTest.SetPowerSave

Intel WLAN Driver does not have the support for PowerSave mode
command. As a WA, return success from the dummy implimentation
in the wpa_supplicant, to make VTS PASS

Tracked-On: OAM-85124
Signed-off-by: Jeevaka Badrappan <jeevaka.badrappan@intel.com>
Signed-off-by: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
---
 src/drivers/driver_nl80211_android.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/drivers/driver_nl80211_android.c b/src/drivers/driver_nl80211_android.c
index ba478888..582f1448 100644
--- a/src/drivers/driver_nl80211_android.c
+++ b/src/drivers/driver_nl80211_android.c
@@ -167,7 +167,10 @@ int wpa_driver_get_p2p_noa(void *priv, u8 *buf, size_t len)
 
 int wpa_driver_set_p2p_ps(void *priv, int legacy_ps, int opp_ps, int ctwindow)
 {
-	return -1;
+	/*return -1;*/
+        /* WA for VTS failure SupplicantStaIfaceHidlTest.SetPowerSave*/
+	return 0;
+
 }
 
 
-- 
2.17.1

